webanno/webanno
# #1460 - Boundary for number features

**What's in the PR**
- add ranges option for number features in the settings UI

**How to test manually**
* Create a new number feature for a span annotation
* Set Limited
* Set minimum and maximum values
* For integer number features use a difference of <= 11 for minimum and maximum
* See rating panel (radio buttons) in the annotation editor panel

**Automatic testing**

**Documentation**
